draft-ietf-scitt-architecture.md

```diff
@@ -126,63 +126,47 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
-Unlike CT, where independent Auditors are responsible for enforcing the consistency of multiple independent instances of the same global Transparency Service, each Transparency Service is required to guarantee the consistency of its own Append-only Log (through the use of a consensus algorithm between replicas of the Transparency Service), but assume no consistency between different Transparency Services.
-
-Breadth of verifier access is critical.
-As a result, the Transparency Service specified in this architecture caters to two types of audiences:
-
-1. **Issuers**: organizations, stakeholders, and users involved in creating or attesting to supply chain artifacts, releasing authentic Statements to a definable set of peers
-1. **Verifiers**: organizations, stakeholders, consumers, and users involved in validating supply chain artifacts, but can only do so if the Statements are known to be authentic.
-Verifiers MAY be Issuers, providing additional Signed Statements, attesting to conformance of various compliance requirements.
-
-The Issuer of a Signed Statement must be authenticated and authorized according to the Registration Policy of the Transparency Service.
-Analogously, Transparent Statement Verifiers rely on verifiable trustworthiness assertions associated with Transparent Statements and their processing provenance in a believable manner.
-If trust can be put into the operations that record Signed Statements in a secure, Append-only Log via online operations, the same trust can be put into the resulting Transparent Statement, issued by the Transparency Services and that can be validated in offline operations.
-
-The Transparency Services specified in this architecture are language independent and can be implemented alongside or within existing services.
-
-The interoperability guaranteed by the Transparency Services is enabled via core components (architectural constituents).
-Many of the data elements processed by the core components are based on the CBOR Signing and Encryption (COSE) standard specified in {{-COSE}}, which is used to produce Signed Statements about Artifacts and to build and maintain an Append-only Log for corresponding Signed Statements.
+The second guarantee is achieved by storing the Signed Statement on an Append-only Log.
+The third guarantee is achieved by implementing the Append-only Log using a verifiable data structure (such as a Merkle Tree {{MERKLE}}).
+
+SCITT is a generalisation of Certificate Transparency {{-CT}}, which can be interpreted as a transparency architecture for the supply chain of X.509 certificates.
+CAs (Issuers) sign X.509 TBSCertificates (Artifacts) to produce X.509 certificates (Signed Statements).
+CAs then submit the certificates to one or more CT logs (Transparency Services).
+The CT logs produce Signed Certificate Timestamps (Transparent Statements).
+The SCTs are checked by browsers (Verifiers), and the Append-only Log can be checked by Auditors.
```

Comment:
```suggestion
* The SCTs are checked by browsers (Verifiers), and the Append-only Log can be checked by Auditors.

Unlike CT, where independent Auditors are responsible for enforcing the consistency of multiple independent instances of the same global Transparency Service, each Transparency Service is required to guarantee the consistency of its own Append-only Log (through the use of a consensus algorithm between replicas of the Transparency Service), but assume no consistency between different Transparency Services.
```

Reply:
> Unlike CT, where independent Auditors are responsible for enforcing the consistency of multiple independent instances of the same global Transparency Service, each Transparency Service is required to guarantee the consistency of its own Append-only Log (through the use of a consensus algorithm between replicas of the Transparency Service), but assume no consistency between different Transparency Services.

I already asked about this sentence [on the list](https://mailarchive.ietf.org/arch/msg/scitt/1oW4DRFFb1g230VmZVQVgj_Mdw8/) because it doesn't make sense to me.

> multiple independent instances of the same global Transparency Service,

I thought every CT log was a single TS? I.e. there are many independent TS, not "the same global" one.

> each Transparency Service is required to guarantee the consistency of its own Append-only Log

This seems to be true for both CT and SCITT? In both cases we hope that the tree is correctly constructed, but if we want to verify it, the auditors can check the consistency proofs.

> through the use of a consensus algorithm between replicas of the Transparency Service

From [Orie's reply](https://mailarchive.ietf.org/arch/msg/scitt/8IwqR-mw8lKwUHg5YT04CIlGrgQ/), this seems to be an implementation detail of a given TS implementation.

This whole confusing sentence is what prompted me to ask about the CT--SCITT mapping in the first place :)
What are "instances", what are "replicas", what's the difference? Again, see [the list](https://mailarchive.ietf.org/arch/msg/scitt/1oW4DRFFb1g230VmZVQVgj_Mdw8/).

If anything, we could add a sentence that CT consists of many *independent* TS, and that Verifiers may want to query all of them? Or there would need to be a federation mechanism, which isn't fully spec'ed I believe? Or maybe I still understand this wrong?

Reply:
I suggest we remove an reference to concrete deployment models from the architecture.

The transparency service is a single logical entity that maintains a log.

how that is implemented behind the scenes is not relevant to the architecture.